src/main/java/org/folio/circulation/services/EventPublisher.java

```diff
@@ -193,7 +197,7 @@ public EventPublisher(PubSubPublishingService pubSubPublishingService) {
 
     var loan = renewalContext.getLoan();
 
-    publishDueDateChangedEvent(loan, loan.getUser());
+    publishDueDateChangedEvent(loan, loan.getUser(), true);
```

Comment:
From my point of view it is better to use additional publish method without true/false flag.

Reply:
I think it would need some code duplication in case of additional method. Renewal context requires the same logic.